### DISCUSSION TITLE
Ram and variable

### DISCUSSION THREAD

**Discussion by Myckel1** [itzg/docker-minecraft-server#2119] [Q&A]
### Hello, I am really not good in docker, I created a minecraft server with a friend, but we want to allocate more ram as well as some environment variable such (HeapRegionSize--ReservePercent///InitiatingHeapOccupancyPercent) etc... I found this about server configuration in paper https://docs.papermc.io/paper/aikars-flags

We have a server with 128gb of ram and 2 processors
So we want to add this shipping variable (java -Xms64G -Xmx64G -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=40 -XX:G1MaxNewSizePercent=50 -XX:G1HeapRegionSize=16M -XX:G1ReservePercent=15 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=20 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTeringnuThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs/ -Daikars.new.flags=true -jar paper.jar --nogui)

Can you please explain to me how to add this variable to the server? all help is greatly appreciated

**Comment by itzg** [comment]
Take a look at
- https://github.com/itzg/docker-minecraft-server#memory-limit
- https://github.com/itzg/docker-minecraft-server#enable-aikars-flags
